src/Controller/CRUDController.php

```diff
@@ -701,14 +712,23 @@ public function showAction($id = null)
 
         $this->admin->setSubject($object);
 
+        /** @var $fields FieldDescriptionCollection */
+        $fields = $this->admin->getShow();
+
+        if (!is_array($fields->getElements()) || 0 === $fields->count()) {
+            throw new \RuntimeException(
+                'No field to show. Did you forget to implement the "configureShowFields" method?'
```

Comment:
This is a BC break, just checked and `configureShowFields` doesn't throw an exception currently if there are no fields.

Reply:
How i can fix it?

Reply:
instead of exception, you throw a deprecation here and then on master you submit another PR but instead of depreciation there you will throw an exception